balboa-agent/README.md

```diff
@@ -120,6 +120,14 @@ Standard SBT Tests.
 sbt balboa-agent/test
 ```
 
+### Integration Tests
+
+sbt balboa-agent/run "-Dbalboa.config=balboa-agent/src/it/resources/config/integrationtest-config.properties"
```

Comment:
Oops. I originally had this, but realized it was the same as the one in the balboa-agent/src directory. So I deleted the new one and used the existing one, but I forgot to change the docs. Updated.